icevision/backbones/resnest_fpn.py

```diff
@@ -0,0 +1,120 @@
+__all__ = [
+    "resnest50",
+    "resnest101",
+    "resnest200",
+    "resnest269",
+]
+
+from icevision.imports import *
+from icevision.utils import *
+from icevision.backbones.resnet_fpn import patch_param_groups
+from torchvision.ops.feature_pyramid_network import LastLevelMaxPool
+from torchvision.models.detection.backbone_utils import (
+    resnet_fpn_backbone,
+    BackboneWithFPN,
+)
+import resnest.torch
```

Comment:
We should make this a soft import, should I work on it?

Reply:
That would be great!